component/src/main/java/org/wso2/extension/siddhi/io/googlepubsub/source/GooglePubSubMessageReceiver.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright (c) 2019  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.io.googlepubsub.source;
+
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.pubsub.v1.PubsubMessage;
+import org.wso2.siddhi.core.stream.input.source.SourceEventListener;
+
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.ReentrantLock;
+
+/**
+ * GooglePubSub Message Receiver.
+ */
+public class GooglePubSubMessageReceiver implements MessageReceiver {
+
+    private SourceEventListener sourceEventListener;
+    private boolean paused;
+    private ReentrantLock lock;
+    private Condition condition;
+
+    public GooglePubSubMessageReceiver(SourceEventListener sourceEventListener) {
+
+        this.sourceEventListener = sourceEventListener;
+        lock = new ReentrantLock();
+        condition = lock.newCondition();
+    }
+
+    @Override
+    public void receiveMessage(PubsubMessage pubsubMessage, AckReplyConsumer ackReplyConsumer) {
+
+        if (paused) {
+            lock.lock();
+            try {
+                condition.await();
+            } catch (InterruptedException ie) {
+                Thread.currentThread().interrupt();
+            } finally {
+                lock.unlock();
+            }
+        }
+
+        ackReplyConsumer.ack();
+        sourceEventListener.onEvent("message :" + "\"" + pubsubMessage.getData().toStringUtf8() + "\"" + " ",
+                null);
+    }
+
+    void pause() {
+
+        paused = true;
+    }
+
+    void resume() {
+
```

Comment:
Please remove unwanted space

Reply:
when we run wso2 code-style that line added automatically.